tensorflow_datasets/testing/bccd.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import PIL
```

Comment:
Consider importing this from ```tfds.core.lazy_imports```. Read more here: https://www.tensorflow.org/datasets/api_docs/python/tfds/core/lazy_imports